src/figures/figureB1.jl

```diff
@@ -179,11 +163,22 @@ function PlotSynvFcrExpr()
 end
 
 function figureB1()
-    p1 = plotIsobologram(2, 3)
-    p2 = plotIsobologramTwo(2, 4)
+    p1 = plotCellIsobologram(2, 3, 2; L0 = 1e-8, f = 24, murine = false, ex = true)
+    p2 = plotCellIsobologram(2, 4, 2)
     p3 = PlotSynGraph()
     p4 = PlotSynValency()
     p5 = PlotSynvFcrExpr()
 
     draw(SVG("figureB1.svg", 1000px, 800px), gridstack([p1 p2 p3; p4 p5 p5]))
 end
+
+function figureS(Cellidx; L0 = 1e-9, f = 4, murine = true)
+    p1 = plotCellIsobologram(1, 2, Cellidx; L0 = L0, f = f, murine = murine)
```

Comment:
If I change this line to plotCellIsobologram(1, 1, ...) It gives the error "Argument Error: collection must be non-empty" This traces back to calculateIsobologram() which traces back to polyfc() and this is where I get lost on what is breaking. @aarmey Hopefully fixing this will also allow figureS3 and S5 to work as well on this pull request.